### PR TITLE
Delete lecture router

### PR DESCRIPTION
**Pull request by meganle**
- DELETE router for lectures
- Added folders for routers
- Small syntax fixes

### CODE REVIEW

File: src/routers/Organizaitons/GetOrganizationsRouter.js [hunk 1]
```diff
@@ -2,34 +2,32 @@
...
+      .paginateOrganization(pageInfo.cursor || 0, pageInfo.count);
 
     return orgs
       // filters falsy entries out
       .filter(Boolean)
       .map(org => ({
         node: {
           id: org.id,
-          name: org.name,
+          name: org.name
```
Comment (mrkev) [inline src/routers/Organizaitons/GetOrganizationsRouter.js]: ,

File: src/routers/Organizaitons/GetOrganizationsRouter.js [hunk 2]
```diff
@@ -2,34 +2,32 @@
...
         },
-        cursor: org.createdAt.valueOf(),
+        cursor: org.createdAt.valueOf()
```
Comment (mrkev) [inline src/routers/Organizaitons/GetOrganizationsRouter.js]: trailing comma

File: src/routers/Lectures/GetLecturesRouter.js [hunk 3]
```diff
@@ -29,9 +27,9 @@ class GetLecturesRouter extends AppDevEdgeRouter<APILecture> {
           id: lecture.id,
           dateTime: lecture.dateTime
         },
-        cursor: lecture.createdAt.valueOf(),
+        cursor: lecture.createdAt.valueOf()
```
Comment (mrkev) [inline src/routers/Lectures/GetLecturesRouter.js]: trailing comma

File: src/routers/Lectures/GetLecturesRouter.js [hunk 4]
```diff
@@ -29,9 +27,9 @@ class GetLecturesRouter extends AppDevEdgeRouter<APILecture> {
           id: lecture.id,
           dateTime: lecture.dateTime
```
Comment (mrkev) [inline src/routers/Lectures/GetLecturesRouter.js]: oh, actually add one here too lol

File: src/routers/Courses/GetCoursesRouter.js [hunk 5]
```diff
@@ -1,36 +1,34 @@
...
       .filter(Boolean)
       .map(course => ({
         node: {
           id: course.id,
           name: course.name,
-          term: course.term,
+          term: course.term
         },
-        cursor: course.createdAt.valueOf(),
+        cursor: course.createdAt.valueOf()
```
Comment (mrkev) [inline src/routers/Courses/GetCoursesRouter.js]: ,

File: src/routers/Courses/GetCoursesRouter.js [hunk 6]
```diff
@@ -1,36 +1,34 @@
...
+      .paginateCourseByOrgId(id, pageInfo.cursor, pageInfo.count);
 
     return courses
       .filter(Boolean)
       .map(course => ({
         node: {
           id: course.id,
           name: course.name,
-          term: course.term,
+          term: course.term
```
Comment (mrkev) [inline src/routers/Courses/GetCoursesRouter.js]: ,

**Comment by meganle** [comment]
Gotcha I'm using eslint and it doesn't like trailing commas lol but I see your point

**Pull request merged**